eo-runtime/pom.xml

```diff
@@ -104,6 +110,34 @@ SOFTWARE.
       <!-- version from parent POM -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.yegor256</groupId>
+      <artifactId>jping</artifactId>
+      <!-- version from parent POM -->
```

Comment:
@yegor256 all dependencies in `eo-runtime` pom.xml must have "test" scope. Build fails otherwise

Reply:
@maxonfjvipon fixed in [aa6a32a](https://github.com/objectionary/eo/pull/2725/commits/aa6a32af380c450a38599a85928e93456b928649)